File: Components/BlogML/Xml/BlogMLAttachment.vb
Imports System.Xml.Serialization

Namespace BlogML.Xml
 <Serializable>
 Public NotInheritable Class BlogMLAttachment

  <XmlAttribute("embedded")>
  Public Property Embedded As Boolean = False

  <XmlAttribute("url")>
  Public Property Url As String

  <XmlAttribute("path")>
  Public Property Path As String

  <XmlAttribute("mime-type")>
  Public Property MimeType As String

  <XmlText>
  Public Property Data As Byte()

 End Class
End Namespace

File: Server/Core/BlogML/Xml/BlogMLAttachment.cs
using System;
using System.Xml.Serialization;

namespace DotNetNuke.Modules.Blog.BlogML.Xml
{
  [Serializable]
  public sealed class BlogMLAttachment
  {

    [XmlAttribute("embedded")]
    public bool Embedded { get; set; } = false;

    [XmlAttribute("url")]
    public string Url { get; set; }

    [XmlAttribute("path")]
    public string Path { get; set; }

    [XmlAttribute("mime-type")]
    public string MimeType { get; set; }

    [XmlText]
    public byte[] Data { get; set; }

  }
}